### DISCUSSION TITLE
North and South routes

### DISCUSSION THREAD

**Discussion by nasserkessas** [Pokemon-PythonRed/Pokemon-PythonRed#101] [Q&A] [answered]
Hello @TurnipGuy30, I was wondering why locations like Route 1 and Viridian City in PPR have a north and south section? I was checking out what's in Viridian North and realised that it doesn't actually exist in the original game. Also: What were you planning to put in Viridian North?

**Comment by riverpaper** [comment, marked as answer]
I'm using Viridian's areas to block the gym, etc. before it should be accessed.
In the same kind of way, Route 1's areas are to provide an encounter zone.

**Reply by riverpaper** [reply]
I will likely make Route 1 into one area sometime in the future.
